resolwe/flow/models/data.py

```diff
@@ -563,42 +561,8 @@ def save(self, render_name=False, *args, **kwargs):
             if "update_fields" in kwargs:
                 kwargs["update_fields"].append("size")
 
-        # Input Data objects are validated only upon creation as they can be deleted later.
-        skip_missing_data = not create
-        validate_schema(
-            self.input, self.process.input_schema, skip_missing_data=skip_missing_data
-        )
-
         render_descriptor(self)
 
-        if self.descriptor_schema:
-            try:
```

Comment:
You have to leave the validation of the descriptor schema here because it can be changed by the user at any time.

Reply:
OK.